### DISCUSSION TITLE
Am I right to say that helloactivity is based on the hello world example from Java and Go SDKs?

### DISCUSSION THREAD

**Discussion by simkimsia** [firdaus/temporal-python-sdk#4] [Q&A]
The discussion comment @firdaus  wrote in https://github.com/firdaus/temporal-python-sdk/discussions/1#discussioncomment-265772 is based on the same hello world example in Java SDK?

https://docs.temporal.io/docs/java-hello-world
https://github.com/temporalio/hello-world-project-template-java

Also I notice there are lots of examples in https://github.com/temporalio/samples-java

Is it cool I try to build something similar to samples-java for python as a way to learn the SDK and temporal.io inner workings as well?

**Comment by firdaus** [comment]
That sounds great.

There's a bit of a barrier to entry to contributing because the SDKs are quite complicated -- any help reducing this is greatly appreciated.

**Reply by simkimsia** [reply]
Thanks

I did ask this in the main temporal forum. I still want to ask you as well.

In case i get stuck with understanding the Java examples I hope I can consult with you, @firdaus 

Whatever you know is fine. Every bit helps.

Thank you
